definitions/npm/ramda_v0.26.x/flow_v0.76.x-v0.103.x/ramda_v0.26.x.js

```diff
@@ -1138,7 +1138,7 @@ declare module ramda {
 
   declare function transpose<T>(xs: Array<Array<T>>): Array<Array<T>>;
 
-  declare function uniq<T>(xs: Array<T>): Array<T>;
+  declare function uniq<A, As: $ReadOnlyArray<A>>(As): As;
```

Comment:
This will need to be corrected too re: input/output types.

Reply:
Done